packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -1,4 +1,7 @@
 <div data-ui-view="dashboard" class="container-fluid mb-5 px-0">
+    <cui-message-container
```

Comment:
There's already a slot where you can hook into to display messages on the dashboard.
see: https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/dedicated/server/server.html#L151

It rely on the `ovh-alert` directive which is located here: 
https://github.com/ovh/manager/blob/master/packages/components/ng-ovh-utils/src/alerter/alerter-directive.js

Using it will helps us to move to another mechanism regarding the way that how we display messages on the control panel.

As you probably noticed, there're also several slots that can display messages on the dashboard.
https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/dedicated/server/server.html#L76-L151

Reply:
yes it's better, i have done with your trick. thanks a lot :)